boot/build.boot

```diff
@@ -0,0 +1,11 @@
+(def project 'boot-clj-lambda/boot-clj-lambda)
+
+(set-env! :dependencies '[[clj-lambda "0.5.0"]]
```

Comment:
And now while testing this boot plugin I had to release `0.5.2` because I found still one issue with dependencies. Could you update to latest? Sorry about this.